Examples/Volume/VolumeMapperBlendModes/index.js

```diff
@@ -50,13 +52,54 @@ ofun.addPoint(-637.62, 0.1);
 ofun.addPoint(700, 0.5);
 ofun.addPoint(3071, 0.9);
 
-const ctfun = vtkColorTransferFunction.newInstance();
-ctfun.addRGBPoint(-3024, 0, 0, 0);
-ctfun.addRGBPoint(-637.62, 0, 0, 0);
-ctfun.addRGBPoint(700, 1, 1, 1);
-ctfun.addRGBPoint(3071, 1, 1, 1);
+const radonParameters = {
+  min: 1530,
+  max: 1030,
+  maxValue: 0.017,
+  minValue: 0.092,
+  unit: 0.3,
+};
+
+function updateColorFunction() {
+  const ctfun = vtkColorTransferFunction.newInstance();
+
+  if (currentBlendMode === 5) {
+    ctfun.addRGBPoint(-3024, 0.0, 0.0, 0.0);
+    ctfun.addRGBPoint(
+      radonParameters.min,
+      radonParameters.minValue,
+      radonParameters.minValue,
+      radonParameters.minValue
+    );
+    ctfun.addRGBPoint(
+      radonParameters.max,
+      radonParameters.maxValue,
+      radonParameters.maxValue,
+      radonParameters.maxValue
+    );
+    ctfun.addRGBPoint(
+      3071,
+      radonParameters.maxValue,
+      radonParameters.maxValue,
+      radonParameters.maxValue
+    );
```

Comment:
please move the CTF creation into a convenient helper function (in the class that has setBlendMode)

Reply:
I'm also surprised that those radonParameters have no impact on `ofun`

Reply:
Why is it resolved but I do not see a convenient helper function ?

Reply:
I added a helper function : createCTF()

Reply:
my bad, I meant to add a convenient `createRadonTransformTransferFunctions(...)` static helper function in `VolumeMapper` that returns both a CTF and PiecewiseFunction